ci-operator/step-registry/ovn/install-hybrid/install/ovn-install-hybrid-install-commands.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+if [[ -z "$RELEASE_IMAGE_LATEST" ]]; then
+  echo "RELEASE_IMAGE_LATEST is an empty string, exiting"
+  exit 1
+fi
+
+cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
+export SSH_PRIV_KEY_PATH=${cluster_profile}/ssh-privatekey
+export PULL_SECRET_PATH=${cluster_profile}/pull-secret
+export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${RELEASE_IMAGE_LATEST}
+export OPENSHIFT_INSTALL_INVOKER=openshift-internal-ci/${JOB_NAME_SAFE}/${BUILD_ID}
+export HOME=/tmp
+
+case "${CLUSTER_TYPE}" in
+aws) export AWS_SHARED_CREDENTIALS_FILE=${cluster_profile}/.awscred;;
+azure4) export AZURE_AUTH_LOCATION=${cluster_profile}/osServicePrincipal.json;;
+gcp) export GOOGLE_CLOUD_KEYFILE_JSON=${cluster_profile}/gce.json;;
+vsphere) ;;
+*) echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"
+esac
+
+dir=/tmp/installer
+mkdir "${dir}/"
+#cp "${SHARED_DIR}/installer" "${dir}/"
+
+# move private key to ~/.ssh/ so that installer can use it to gather logs on
+# bootstrap failure
+mkdir -p ~/.ssh
+cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
+
+
+# testing putting the create manifests and stuff here.
+cp "${SHARED_DIR}/install-config.yaml" "${dir}/"
+
+TF_LOG=debug openshift-install --dir="${dir}" create manifests
+
+cat >> "${dir}/manifests/cluster-network-03-config.yml" << EOF
```

Comment:
I would much rather see you break apart the current install step into a manifest-create and a run-install or something. Whenver we find ourselves copy-pasting in this system we _need_ to be refactoring to make this reusable

Reply:
updated with the code split into steps